src/Console/Shell.php

```diff
@@ -197,6 +198,8 @@ public function __construct(ConsoleIo $io = null)
         if (isset($this->modelClass)) {
             $this->loadModel();
         }
+
+        $this->loadTasks();
```

Comment:
Why does this need to be done earlier? Wouldn't doing this in `initialize()` still be early enough?

Reply:
I guess it would be.

Reply:
I haven't realized it's already been called there. Change the code. I've also refactored the added code a little and put it in it's own method to allow easier overriding and reduce method complexity.